### ISSUE TITLE
[CST] [ENG] Claim updates on the status page

### ISSUE THREAD

**Issue by uxgary** [department-of-veterans-affairs/va.gov-team#82375]
## Value Statement
**_As a_** Veteran who prefers to see claim details at a glance,
**_I want to_** be see a snapshot of my claim details on the status page
**_So that_** I can see pertinent information like current step and brief description without seeing the full claim details

---
## Background Context
As part of improving the claim phase experience, we aim to increase transparency on the background process of claims. To start, we are expanding the claims steps from 5 to 8 steps and providing a brief description about each step. Additionally, we are providing claims updates so Veterans can have a better sense of where they are in the claims process. 

For additional information see resources listed below: 

- [Collaboration Cycle for Benefits team 1, Claim status tool , Claim Phase Experience](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71293#top)

<br>

Status page | Overview page <br> (title reference)|
--- | --- |  
<img width="341" alt="Screenshot 2024-05-06 at 11 40 01 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/13239903/9c864b90-f0f3-42d8-a9c8-7d4e86dbc883"> | <img width="268" alt="Screenshot 2024-05-20 at 4 07 41 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/13239903/b1a45dec-d05b-462f-afaf-7392270e23d3"> |

### Related Issue(s)
- department-of-veterans-affairs/va.gov-team/issues/82376

### New Feature
- Why are we doing this work?
- [Epic Link, Expand Claim Phases](https://github.com/department-of-veterans-affairs/va.gov-team/issues/82360)

## Design
- Ensure revised mocks are ready for engineers
- [Figma file](https://www.figma.com/design/bWELjWsVqkf3BDx0PXqqam/Claim-Contextualization?node-id=1924-38524&t=ouEH8MXj79IYrcze-4) for visual reference
- [Figma design for Overview accordion header](https://www.figma.com/design/bWELjWsVqkf3BDx0PXqqam/Claim-Contextualization?node-id=1963-23774&t=rjzdcqKakC4AVE08-4)
- [Content file](https://docs.google.com/document/d/1WfWCfoQgKoMBg835VP7j_ptXmqb2o3RVme2u5dMOEr8/edit?usp=sharing) containing the body copy for the status page

### Enablement team (if needed)
@member of team

## Engineering 
- We want the date displayed in the description the phasechangedate
- Will use the field `claim.attributes.claimPhaseDates.latestPhaseType` for the status for the 8 steps

## Out of scope
- Add here

## Open questions
- What does the "Learn more about this step" link go to?

## Tasks
- [x] use the [content doc](https://docs.google.com/document/d/1WfWCfoQgKoMBg835VP7j_ptXmqb2o3RVme2u5dMOEr8/edit?usp=sharing) for the description blurb for each phase on the claim card
- [x] Header title should match the accordion headers on the overview page (ie. Step 2 of 8: Initial review, Step 3 of 8: Evidence gathering)
- [ ] Add new field with text 'Moved to this step on <DATE>'
- [x] "Learn more about this step" link navigates to the overview page

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Design performs design QA and verifies the implementation matches the design spec
- [ ] Accessibility specialist performs accessibility review (in code or design)
- [ ] Engineering identifies staging users required to test and shares account and credentials with design and product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria
- [x] There is header text for claim phases 1-7 in the "What We're Doing" component and the number and step description matches exactly what is used as headers on the Overview page.
- [x] The part of the header that mentions the claim "step" matches exactly the name of the step on the Overview page for the new timeline; When a user is on step 1 of 8 the header shows <this> and the description shows <this>
- [x] The description for each phase matches the "What We're Doing" text from [this doc](https://docs.google.com/document/d/1WfWCfoQgKoMBg835VP7j_ptXmqb2o3RVme2u5dMOEr8/edit#heading=h.55axkr7jidx0) consistent with the phase steps. 
- [x] There is a link to the overview section of CST that says "Learn more about this step"
- [x] There is "moved to" text that clearly calls out the date the claim moved to this step. 
- [x] Behind cst_claim_phases feature flag

**Comment by uxgary** [comment]
I've created a google doc with the content we can use for the [status page content](https://docs.google.com/document/d/1WfWCfoQgKoMBg835VP7j_ptXmqb2o3RVme2u5dMOEr8/edit?usp=sharing).

**Comment by pmclaren19** [comment]
@uxgary this ticket/mocks are lacking information that engineering will need to work  and point this. We need answers to the following things before we can do that...

- This is tickets supposed to be a design and eng ticket or just eng? Noticed you added design tasks for information that is crucial for eng
- If this ticket relies on the google doc that you commented about I think that should be called out in the actual ticket description. Also your google doc is noting what you want in the overview tab not the status tab so confused on the desired changes.
-  There should also be tasks to note what you expect to be changed `EX: When a user is on step 1 of 8 the header shows <this> and the description shows <this>.` Is there anything else other than the header and text that is changing?
- Should this be behind a feature flag?

Let me know if you have any questions about this.

**Comment by pmclaren19** [comment]
Started working this ticket and found that the api call that we use to  currently get the status from only provided the 5 steps and not the 8 steps; `claim.attributes.status`. In order to get the 8 steps we will have to make a request to lighthouse to return  `claim.attributes.claimPhaseDates.latestPhaseType` to us in the api call. Will work with @jerekshoe to make this request to lighthouse.

**Comment by pmclaren19** [comment]
Noticed that the google doc and the design doc had different text for the Step 8 Heading - discussed with @uxgary and he updated the design docs to reflect this update. Slack convo here https://dsva.slack.com/archives/C056854SVT8/p1717016613526739

Updating pr to reflect this change!

**Comment by pmclaren19** [comment]
@uxgary please review for qa design.

**Comment by uxgary** [comment]
Reviewed, this passes.
